### DISCUSSION TITLE
regolith-de, Arch and Gnome update finally broke stuff

### DISCUSSION THREAD

**Discussion by sukulent** [regolith-linux/regolith-desktop#1000] [General]
Hello,

I have been using the regolith arch port from gardotd426 with no big problems (just some i3bar pango syntax journalctl clutter lol) until recently. It is actually surprising how long it managed to work without any updates, but it finally broke.

I have wrote a bit more [here](https://github.com/gardotd426/regolith-de/issues/51). But in a nutshell:
gnome-flashback [updated](https://gitlab.archlinux.org/archlinux/packaging/packages/gnome-flashback/-/commit/d27d613c789c221519174f05a1a91bd7bb32034c) and I got login loop. To get into semi-working session I had to remove the `--builtin` from `/usr/bin/regolith-session` as new gnome complained - the option got removed.

Now the connection between regolith and gnome is not working so mouse settings, touchpad settings, dunst, etc. I am glad I have a working computer but I'd like to fix this, at least temporarily (I really like using regolith), then if I find some time (studying uni) I could try updating the PKGBUILD for new regolith.

I see there is new xdg-desktop-portal-regolith, am I looking right? Did I update too too-bleeding-edge-gnome?

**Comment by kgilmer** [comment]
Hi @sukulent , what version of GNOME are you running now (45?)?

**Comment by sukulent** [comment]
`gnome-shell --version` says I got GNOME Shell 46.0 👀

**Comment by denny-futurice** [comment]
Ah, i only now see this discussion thread. I created a defect report out of it here: https://github.com/regolith-linux/regolith-desktop/issues/1001

I was tempted to push a PR to remove the argument, but I'm not entirely sure what the implications are of removing the argument, I'll try to read the upstream GNOME docs what work around is required.
